server/src/middlewares/auth.js

```diff
@@ -34,6 +34,29 @@ const checkJwt = (req, res, next, prisma) => {
       }`
     )
 
+  const userUpsert = () =>
```

Comment:
Je propose de renommer cette fonction pour clarifier qu'elle n'est utile qu'en cas d'authentification désactivée.

```suggestion
  const upsertNoAuthUser = () =>
```